Palaso.MSBuildTasks/UnitTestTasks/NUnit3.cs

```diff
@@ -11,6 +11,14 @@ namespace Palaso.BuildTasks.UnitTestTasks
 	/// </summary>
 	public class NUnit3 : NUnit
 	{
+		public NUnit3()
+		{
+			// REVIEW: This should probably be true for NUnit also, but changing
+			// the logic there could potentially cause unexpected results for existing
+			// callers whereas the NUnit3 task is new enough, I think we are okay.
+			FailTaskIfAnyTestsFail = true;
```

Comment:
I'm not convinced that we want to always fail on test failures. At least on Jenkins this eliminates the distinction between failing tests (unstable) and failing build (failed). If TC needs this then we should make it conditional on the `TeamCity` property.